controllers/manifest_controller/helper/helper.go

```diff
@@ -21,11 +25,22 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/google/go-containerregistry/pkg/v1/types"
-	. "github.com/onsi/gomega"
+	gmg "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const CredSecretLabelKeyForTest = "operator.kyma-project.io/oci-registry-cred" //nolint:gosec
+var (
+	Cancel                   context.CancelFunc
+	Ctx                      context.Context
+	K8sClient                client.Client
+	Server                   *httptest.Server
+	ErrManifestStateMisMatch = errors.New("ManifestState mismatch")
+	ManifestFilePath         = "../../pkg/test_samples/oci/rendered.yaml"
```

Comment:
I think its rather an anti-pattern to have ctx, client, etc. as shared global variables between multiple test suits.
I would prefer to not to have it as global variables, but the suits them self should each have there own clients etc.

```suggestion
	ErrManifestStateMisMatch = errors.New("ManifestState mismatch")
	ManifestFilePath         = "../../pkg/test_samples/oci/rendered.yaml"
```

Reply:
You are right. At the same time they are global variables in existing suites. This variables are necessary for the functions defined here. We have three options:
- duplicate the entire code between the suites, removing this file (I don't like it)
- have the common functions defined here, but without global variables - make them parameters to the functions (pollutes nice function API with low-level stuff like k8sClient)
- wrap all these params in some struct and make all the functions defined here a methods of this struct. Acceptable, but initially I considered it an over-engineering.

What's your opinion?

Reply:
I would go for the second option 👍🏻 So we could possibly use it in any combination of suits, without the need to think about race conditions